langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaApi.java

```diff
@@ -30,4 +27,12 @@ interface OllamaApi {
     @Headers({"Content-Type: application/json"})
     @Streaming
     Call<ResponseBody> streamingChat(@Body ChatRequest chatRequest);
+
+    @GET("/api/tags")
+    @Headers({"Content-Type: application/json"})
+    Call<ModelListResponse> modelList();
```

Comment:
@bidek could you please add a few integration tests for these 2 new endpoints?

Reply:
Added tests for the new endpoints